src/Abstractions/NexusMods.Abstractions.Loadouts/GameMetadata.cs

```diff
@@ -27,4 +29,33 @@ public partial class GameMetadata : IModelDefinition
     /// </summary>
     public static readonly StringAttribute Path = new(Namespace, "Path") {IsIndexed = true};
     
+    /// <summary>
+    /// The last applied loadout to this game state
+    /// </summary>
+    public static readonly ReferenceAttribute<Loadout> LastSyncedLoadout = new(Namespace, nameof(LastSyncedLoadout)) { IsOptional = true };
+    
+    /// <summary>
+    /// The 'AsOf' transaction ID of the last applied loadout
+    /// </summary>
+    public static readonly ReferenceAttribute<Transaction> LastSyncedLoadoutTransaction = new(Namespace, nameof(LastSyncedLoadoutTransaction)) { IsOptional = true };
+    
+    /// <summary>
+    /// The 'AsOf' transaction ID of the initial state when the game folder was first indexed
+    /// </summary>
+    public static readonly ReferenceAttribute<Transaction> InitialStateTransaction = new(Namespace, nameof(InitialStateTransaction)) { IsOptional = true };
```

Comment:
Since these Transactions are not bound to a specific entity, it should be very explicit what they refer to
```suggestion
    public static readonly ReferenceAttribute<Transaction> InitialDiskStateTransaction = new(Namespace, nameof(InitialStateTransaction)) { IsOptional = true };
```